functions.photon.php

```diff
@@ -271,3 +271,20 @@ function jetpack_photon_banned_domains( $skip, $image_url, $args, $scheme ) {
 
 	return $skip;
 }
+
+
+/**
+ * Jetpack Photon - Support Text Widgets.
+ * 
+ * @access public
+ * @param mixed $content Content from text widget.
+ * @return void
```

Comment:
- `$content` type is `string` instead of `mixed`.
- @return type is `string` instead of `void`